src/main/java/model/Ball.java

```diff
@@ -0,0 +1,19 @@
+package model;
+
+public class Ball {
+    private final int number;
+    private final String color;
+
+    public Ball(int number, String color) {
+        this.number = number;
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color='" + color + '\''
+                + ", number=" + number
+                + '}';
+    }
+}
```

Comment:
Classes, fields, constructors, methods must have access modifiers otherwise default will be used that isn't a good practice. It's better to explicitly define the access level, for example, by marking the `Ball` class as public.